src/modules/systemlib/circuit_breaker_params.c

```diff
@@ -106,7 +105,7 @@ PX4_PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK);
  * @max 121212
  * @group Circuit Breaker
  */
-PX4_PARAM_DEFINE_INT32(CBRK_FLIGHTTERM);
+PARAM_DEFINE_INT32(CBRK_FLIGHTTERM, 121212);
```

Comment:
Does ROS reach in and grab headers somehow? I searched the whole repository and couldn't find a reference to those #defines. Would be good for me to understand the connection.

What do you want me to do here? I can toss the change. Then QGC will lose the default values.

> On Apr 20, 2015, at 10:04 AM, Thomas Gubler <notifications@github.com> wrote:
> 
> In src/modules/systemlib/circuit_breaker_params.c <https://github.com/PX4/Firmware/pull/2043#discussion_r28707451>:
> 
> > @@ -106,7 +105,7 @@ PX4_PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK);
> >   * @max 121212
> >   * @group Circuit Breaker
> >   */
> > -PX4_PARAM_DEFINE_INT32(CBRK_FLIGHTTERM);
> > +PARAM_DEFINE_INT32(CBRK_FLIGHTTERM, 121212);
> mc_att_control(_multiplatform) uses circuit breakers. That's why these params were changed. With this PR the ROS default value will be disconnected from the PX4/nuttx default value but it will continue to compile. The PX4_PARAM_DEFINE_INT32 macro was a workaround to have only 1 definition of the default values.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/PX4/Firmware/pull/2043/files#r28707451>.
>